src/test/resources/data/InstructorFeedbackEditQuestionsUiTest.json

```diff
@@ -0,0 +1,113 @@
+{
+  "accounts": {
+  	"instructorWithSessions": {
+  	  "googleId": "CFeedbackEditUiT.instructor",
```

Comment:
When creating/modifying the json files, take note to change the ids/course ids if necessary so that the data will not conflict/interfere with each other when the tests are run in parallel.